<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Doctor</class>
 <widget class="QWidget" name="Doctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="UpdateInfo">
      <attribute name="title">
       <string>UpdateInfo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="1">
        <widget class="QLineEdit" name="registrationFee"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="photo"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="dailyPatientLimit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>EmployeeID</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="profile"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Profile</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="department"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Photo(URL)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>WorkSchedule</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>RegistrationFee</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="employeeId"/>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="doneBtn">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>DailyPatientLimit</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="workschedule"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Department</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="doctorName"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UpdatePassword">
      <attribute name="title">
       <string>UpdatePassword</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLineEdit" name="oldPass"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="newPass"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="checkPass"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="doneBtn2">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>NewPassword</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>OldPassword</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Check Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AppointmentList">
      <attribute name="title">
       <string>AppointmentList</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="5">
        <widget class="QLineEdit" name="selectName"/>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="selectAppoBtn">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>PatientName</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>PatientIdCard</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>TimeSlot</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Action</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QScrollArea" name="ListScreen">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="widget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>994</width>
            <height>583</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CaseManaging">
      <attribute name="title">
       <string>CaseManaging</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Diagnosis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Prescription</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="diagnosis"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Advice</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="advice"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="prescription"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="OKBtn">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CaseSearching">
      <attribute name="title">
       <string>CaseSearching</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" colspan="5">
        <widget class="QLineEdit" name="selectCase"/>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="selectBtn_2">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>PatientName</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>PatientIDCard</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>Diagnosis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>Prescription</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>Advice</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QScrollArea" name="CaseListScreen">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>994</width>
            <height>583</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
      <zorder>CaseListScreen</zorder>
      <zorder>selectCase</zorder>
      <zorder>selectBtn_2</zorder>
      <zorder>label_33</zorder>
      <zorder>label_34</zorder>
      <zorder>label_35</zorder>
      <zorder>label_36</zorder>
      <zorder>label_37</zorder>
     </widget>
     <widget class="QWidget" name="LeaveManaging">
      <attribute name="title">
       <string>LeaveManaging</string>
      </attribute>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>171</x>
         <y>225</y>
         <width>63</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>EndTime</string>
       </property>
      </widget>
      <widget class="QPushButton" name="leaveReqBtn">
       <property name="geometry">
        <rect>
         <x>247</x>
         <y>301</y>
         <width>591</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Leave Request</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="endTime">
       <property name="geometry">
        <rect>
         <x>247</x>
         <y>225</y>
         <width>591</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="startTime">
       <property name="geometry">
        <rect>
         <x>247</x>
         <y>149</y>
         <width>591</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="leaveCancBtn">
       <property name="geometry">
        <rect>
         <x>247</x>
         <y>378</y>
         <width>591</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Leave Cancel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fetcLastRequestBtn">
       <property name="geometry">
        <rect>
         <x>247</x>
         <y>455</y>
         <width>591</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Fetch Last Request</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>171</x>
         <y>149</y>
         <width>69</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>StartTime</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>医患沟通</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>话题Id</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="topicEnter"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="topicComboBox">
         <item>
          <property name="text">
           <string>按话题Id排序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>按时间排序</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="topicCheckBtn">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>话题Id</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>                                                                          最近发送时间</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QScrollArea" name="topicScreen">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>994</width>
            <height>583</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
